src/backports/zoneinfo/_common.py

```diff
@@ -25,6 +25,19 @@ def load_tzdata(key):
         # UnicodeEncodeError: If package_name or resource_name are not UTF-8,
         #   such as keys containing a surrogate character.
         raise ZoneInfoNotFoundError(f"No time zone found with key {key}")
+    except (IsADirectoryError, PermissionError):
+        # A few exceptions inherited from OSError can be raised in various scenarios:
+        #
+        # IsADirectoryError: If the resource_name links to a directory
+        #   (e.g. Australia)
+        # PermissionError: If the resource_name links to a directory on Windows
+        #   (e.g. Pacific)
+        resource_path = importlib_resources.files(package_name).joinpath(key)
```

Comment:
I'm fairly certain this will fail on Python 3.8. I'm not sure why the test suite isn't executing to let you know this, though.

Reply:
Seems you're correct here. Not at this time sure of a good alternative to what 3.9 does.